doc/developing/misc.md

```diff
@@ -0,0 +1,61 @@
+# Miscellaneous
+
+## Re-sign the Android APK from CI
```

Comment:
Why "re-sign"? Since the APKs from CI are unsigned, shouldn't it be just "sign"?

Thanks for these Android docs btw, they would have helped me a lot.